mak/wildtype.bat

```diff
@@ -0,0 +1,16 @@
+@echo off
+REM Generates makefile sequences for Digital Mars make.exe (it doesn't supports wildcards by itself)
+
+set TEXT=%~1
+set PATH=%2
+set MASK=%3
+set DESTINATION=%~4
+set REMOVE_FIRST_NUM=%5
+set PREFIX=%~6
+
+set WORKING_DIR=%cd%
+set ROUTINE=%cd%\mak\wroutine.bat
+
+C:\Windows\System32\forfiles.exe /p %PATH% /m %MASK% /s /c "cmd /c %ROUTINE% @RELPATH"
```

Comment:
Don't use the absolute path here. I would expect c:\Windows\System32 to be in PATH. Even if not, c:\Windows is not guaranteed to be the installation path, %windir% contains it.

Reply:
At first I tried it without full path, but it works only in local console, but not in .bat files. I don't know why. If you can please fix it.